pulp_2to3_migration/app/migration.py

```diff
@@ -101,6 +104,9 @@ def migrate_repositories(plan):
                         pulp2repo.pulp3_repository = repo
                         pulp2repo.save()
                     else:
+                        if not pulp2repo.pulp3_repository:
+                            pulp2repo.pulp3_repository = repo
+                            pulp2repo.save()
```

Comment:
I think it only covers the case when a repo_version_setup has one repo in it, and when pulp3_repo_name is the same as in pulp2.
See a suggesiton to cover that.
https://github.com/pulp/pulp-2to3-migration/compare/master...goosemania:issue6964